ibis/backends/tests/test_join.py

```diff
@@ -385,3 +385,33 @@ def test_join_conflicting_columns(backend, con):
         }
     )
     backend.assert_frame_equal(result, expected)
+
+
+@pytest.mark.never(
+    [
+        "bigquery",
+        "clickhouse",
+        "datafusion",
+        "druid",
+        "exasol",
+        "flink",
+        "impala",
+        "mssql",
+        "mysql",
+        "oracle",
+        "polars",
```

Comment:
It doesn't have to be a part of this PR, but `polars` almost certainly supports positional joins via `concat`, so this should probably be marked as a `notimpl`

Reply:
Easy enough, I'll just implement it.